elements/standard/exactcpuswitch.hh

```diff
@@ -0,0 +1,59 @@
+#ifndef EXACTCPUSWITCH_HH
+#define EXACTCPUSWITCH_HH
+#include <click/batchelement.hh>
+#include <click/vector.hh>
+CLICK_DECLS
+
+/*
+ * =c
+ * ExactCPUSwitch()
+ *
+ * =s threads
+ *
+ * classifies packets by cpu
+ *
+ * =d
+ *
+ * Can have any number of outputs.
+ * Chooses the output on which to emit each packet based on the thread's cpu. 
+ * Contrary to CPUSwitch, the output is choosed using the thread-vector
+ * mapping. It means that the outputs will be evenly shared among threads.
+ * Eg, with CPUSwitch if threads 1, 7 and 8 push packets to a CPUSwitch with
+ * 3 outputs, the 1 and 7 will push to output 1, and 8 to output 2. Leading to
+ * an imbalance. 
+ *
+ * ExactCPUSwitch will use the known list of possible threads to balance
+ * evenly one input thread per output port.
+ *
```

Comment:
Add documentation about input args and handlers

Reply:
There's not any :p